new/LIT7346QeneLasabaya.xml

```diff
@@ -0,0 +1,80 @@
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Qǝne poem. The type is indicated as mawaddǝs in the manuscript. According to 
+                    <bibl><ptr target="bm:Guidi1900Qene"/><citedRange unit="page">4-5</citedRange></bibl>, it is a 
+                    kʷǝllǝkǝmu mawaddǝs-type with nine verses. The author is not indicated.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Mawaddes"/>
+                    <term key="Kwellekomu"/>
+                </keywords>        
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2025-04-22">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>                
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="PetermannIINachtrag24"/>, fol. 7ra. Division into verses is indicated 
+                    with ። .</note>
+                <ab>
+                    መወድስ፡
+                    <l n="1">ለሰብናየ፡ ቤተ፤ ን<add place="margin">ጉሥ</add><supplied reason="omitted">፡</supplied> ሀልዮ፡ 
+                        <choice resp="CH"><sic>ስ</sic><corr>ሰ</corr></choice>ርዌሁ፡ ዘኢይሰፈር፡ ምጠቁ።</l> 
+                    <l n="2">አምጣነ፡ ብዙሕ፡ ኮነ፡ ትካዘ፡ ዓለም፡ ስዋቁ።</l>
+                    <l n="3">ለማእምራን፡ ኢይትአምር፡ ኍልቁ።</l>
+                    <l n="4">እስከ፡ መጠነዝ፡ ይትላጸቁ።</l> 
+                    <l n="5">ኢያድ<choice resp="CH"><sic>ኃ</sic><corr>ኅ</corr></choice>ንዎ፡ ለምንት፡ እምነፋስ፡ ንጼት፡ ለለ፡ ሠርቁ።</l>
+                    <l n="6">እንዘ፡ ይገፈትዖሙ፡ ከመ፡ በዓቢይ፡ ይደቁ።</l>
+                    <l n="7">ምስሌሁ፡ በኣሕብሮ፡ ዕለ፡ ርእይዎ፡ እስከ፡ ይለህቁ።</l>
```

Comment:
maybe, to consider ይስሀቁ as the last word in line 7, I do not remember if Jussive is possible after እስከ, I think the shape of the letter is closer to ሰ (6 order) and ለ.